prow/git/git.go

```diff
@@ -155,7 +155,7 @@ func (c *Client) Clone(repo string) (*Repo, error) {
 		if err := os.MkdirAll(filepath.Dir(cache), os.ModePerm); err != nil && !os.IsExist(err) {
 			return nil, err
 		}
-		remote := fmt.Sprintf("%s/%s", base.Path, repo)
+		remote := fmt.Sprintf("%s/%s", base.String(), repo)
```

Comment:
Can you add a unit test that covers this change. Also what happens when the path set on base is `/` ?